include/lib/aarch64/arch.h

```diff
@@ -186,6 +186,11 @@
 #define HCR_IMO_BIT		(1 << 4)
 #define HCR_FMO_BIT		(1 << 3)
 
+/* ISR definitions */
+#define ISR_A_SHIFT		8
+#define ISR_I_SHIFT		7
+#define ISR_F_SHIFT		6
+
```

Comment:
It is a mnemonic for whom is debugging such an abort with the help of ISR_EL1. I could remove it if you still don't want it, don't you?